inc/admin/menus/class-sidebar.php

```diff
@@ -67,6 +73,20 @@ public function hooks(): void {
 		remove_action( 'admin_init', '\Pressbooks\Admin\NetworkManagers\restrict_access' );
 	}
 
+	public function removePatternsSubMenuItem(): void {
+		remove_submenu_page( 'themes.php', 'edit.php?post_type=wp_block' );
+	}
+
+	public function restrictPatternsPageAccess(): void {
+		global $pagenow;
+
+		if ( $pagenow !== 'edit.php' || ! isset( $_GET['post_type'] ) || $_GET['post_type'] !== 'wp_block' ) {
+			return;
+		}
+
+		wp_die( __( 'Sorry, you are not allowed to access this page.', 'pressbooks' ), 403 ); // phpcs:ignore Pressbooks.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Lint issue:
```bash
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 87 | ERROR | All output should be run through an escaping function
    |       | (see the Security sections in the WordPress Developer
    |       | Handbooks), found '__'.
    |       | (Pressbooks.Security.EscapeOutput.OutputNotEscaped)
----------------------------------------------------------------------
```
Here we have 2 options:
- ignore phpcs
- Use an escaping data method, see: https://developer.wordpress.org/apis/security/escaping/. I don't think any of the would make sense in this case

Reply:
I don't see the point to escape this string since it does not have variables nor html, so ignoring phpcs makes more sense to me.